website/api/controllers/deliver-apple-csr.js

```diff
@@ -25,7 +25,8 @@ module.exports = {
     },
 
     invalidEmailDomain: {
-      description: 'This email address is on a denylist of domains and was not delivered.'
+      description: 'This email address is on a denylist of domains and was not delivered.',
+      statusCode: 400
```

Comment:
😘 beautiful!

Minor nit:
```
responseType: 'badRequest'
```

^ instead of statusCode

that'll take care of content negotiation just in case someone somehow hits this w/ a weird client